dev-php/promise/promise-2.8.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Promises/A implementation for PHP"
+HOMEPAGE="https://github.com/reactphp/promise/"
+SRC_URI="https://github.com/reactphp/promise/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-lang/php:*
+	dev-php/fedora-autoloader"
+
```

Comment:
Would it be possible to add test for this package ?

Reply:
Haven't really tried, just tested by running composer itself. They are not included in source code anyway.

Reply:
Yes tests are not in the release tarball :(